source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -87,6 +87,23 @@ Settings
        | **Default**: ``""``
        | **Accepted Values**: ``""``, ``"updateLookup"``, ``"whenAvailable"``, or ``"required"``
        
+   * - | **change.stream.show.expanded.events**
```

Comment:
I noticed some fields in the left column wrap, on staging. Can we widen the left column? This is an optional comment.